mizar/common/common.py

```diff
@@ -434,6 +435,36 @@ def conf_list_has_max_elements(conf, conf_list):
         return True
     return False
 
+def support_offload_xdp_itf_names():
+    """
+    According to the list of NIC names, corresponding logic interface names are returned.
+    """
+    logical_itf_names = []
+    with open("/var/mizar/supported_xdp_offload_nics.yaml", "r", encoding="utf-8") as f:
+            supported_nic_names = yaml.load(f, Loader=yaml.FullLoader)
+
+    ret, data = run_cmd("ls -l /sys/class/net")
+    if ret is not None:
```

Comment:
This seems very complicated. Can we not just use the lspci command with -mm or -nn flag, then filter on 'Ethernet controller' ?